frontend/components/Header.js

```diff
@@ -1,9 +1,55 @@
 import React from 'react';
+import styled from 'styled-components';
+<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet"></link>
```

Comment:
Try moving line 3 to the Header area! most `link hrefs` should be in the meta tags of a project :)